stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -32,6 +45,10 @@
 
   private final Context mContext;
 
+  private Map<String, AsyncPrettyPrinter> mRequestIdMap = Collections.synchronizedMap(
```

Comment:
You need to be using `final` in cases like this.  Please repeat this advice all over the diff.